### DISCUSSION TITLE
combination filters (where my fzf AND at?)

### DISCUSSION THREAD

**Discussion by Liquidmantis** [nvim-telekasten/telekasten.nvim#47] [Q&A] [answered]
Since I won't have time to dig into this for a bit, I'm posting here to see if anyone has recommendations.  

Admittedly this is probably more of a telescope question, but I'm trying to figure out a way to chain filters.  For example, maybe I want to find notes that are tagged `#work` and have a `#todo` tag or the `todo` string in them.  I don't think fzf supports this logic, and it certainly can't be done with linewise regex.  

My suspicion is that this would require feeding the results of a Telescope search into another one, which could potentially offer a repeat loop.  That's the solution I think I'll pursue later for a handful of reasons, but maybe there's an easy interim solution, like some sort of custom ripgrep query I can use.

**Comment by Liquidmantis** [comment]
Backgrounding in meetings is fun…. I think I might be waiting for this https://github.com/nvim-telescope/telescope.nvim/pull/1115

**Comment by Liquidmantis** [comment, marked as answer]
Absent a builtin refine/chain feature in Telescope, I'm now solving my need by using `<C-q>` in Telescope to send results to the quickfix list, then my hotkey for `:Telescope quickfix` to grep the quickfix entries.  It's not ideal replacing the quickfix list contents, but the times it would cause me issues are likely few and far between.

**Comment by renerocksai** [comment]
This is related to the search feature (for persistent search aka saved searches) where you can use a minimal query language like `#work and #todo` or `#work and 'todo'`. 

I don't know how fast it would update, but I can imagine putting the query string in the Telescope prompt and re-running the search upon change - which is not a refining of search results but a re-issuing of a global search.
